test/test_cnvlib.py

```diff
@@ -16,24 +16,42 @@
 from skgenome import GenomicArray, tabio
 
 import cnvlib
-from cnvlib import cnary, core, fix, smoothing, vary
+from cnvlib import cnary, core, fix, smoothing, vary, read
 
 
 class CNATests(unittest.TestCase):
     """Tests for the CopyNumArray class."""
 
     def test_empty(self):
         """Instantiate from an empty file."""
-        cnarr = cnvlib.read("formats/empty")
+        cnarr = cnvlib.read("formats/empty", None)
```

Comment:
Could the second argument be handled by a default value of None in the signature of `cnvlib.read` instead?